src/api/app/models/token/workflow.rb

```diff
@@ -9,10 +9,10 @@ def self.token_name
 
   def call(options)
     set_triggered_at
+    @scm_webhook = options[:scm_webhook]
 
-    raise Token::Errors::MissingPayload, 'A payload is required' if options[:payload].nil?
+    raise Token::Errors::MissingPayload, 'A payload is required' if @scm_webhook.nil?
```

Comment:
I don't think this is right, since `@scm_webhook` contains the payload. There's also a similar validation happening in the controller. We can clean this up later. For now I suggest going with:

```suggestion
    raise Token::Errors::MissingPayload, 'A payload is required' if @scm_webhook.payload.nil?
```

Reply:
Right, it's updated.